lib/rails_transactional_outbox/record_processors/active_record_processor.rb

```diff
@@ -15,7 +15,12 @@ def applies?(record)
       end
 
       def call(record)
-        model = record.infer_model or raise CouldNotFindModelError.new(record)
+        model = record.infer_model
+        if model.nil?
+          return unless RailsTransactionalOutbox.configuration.raise_not_found_model_error
```

Comment:
using `if` is more readable, how about doing it the other way round like:
```
raise CouldNotFindModelError.new(record) if raise_not_found_model_error?

return
```

Reply:
raise with condition happens to be a long line :) `unless` is pretty readable for simple conditions like that. Anyway adjusted